examples/ch32v305/riscv32imfc-unknown-none-elf.json

```diff
@@ -0,0 +1,19 @@
+{
+  "arch": "riscv32",
+  "atomic-cas": false,
+  "cpu": "generic-rv32",
+  "crt-objects-fallback": "false",
+  "data-layout": "e-m:e-p:32:32-i64:64-n32-S128",
+  "eh-frame-header": false,
+  "emit-debug-gdb-scripts": false,
+  "features": "+m,+f,+c,+forced-atomics",
```

Comment:
`+forced-atomics` is actually in every single `none` target. Which means `core::atomic::*` actually exists.